pyproject.toml

```diff
@@ -59,6 +59,9 @@ packaging = "^23.1"
 pipdeptree = "^2.13.0"
 reflex-hosting-cli = ">=0.1.2"
 charset-normalizer = "^3.3.2"
+wheel = "^0.42.0"
```

Comment:
This is required for the build backend. As discussed, this is also already in the install payload. I would prefer to explicitly include this in case other packages depending on wheel get removed and we'll get unexpected failure.